iosApp/fastlane/Managers/TestManager.rb

```diff
@@ -15,6 +15,7 @@ def build_and_test(scheme:, targets:)
       code_coverage: true,
       result_bundle: true,
       only_testing: targets,
+      number_of_retries: 2,
```

Comment:
UITest is now using real network and sometimes that can fail randomly, this is added so that we don't have to rebuild the test again by hand and reduce the total build time.